pages/Sugars.tsx

```diff
@@ -0,0 +1,76 @@
+import Row from "../components/calculator/rows/row";
+import Header from "../components/header/header";
+import Footer from "../components/footer/footer";
+import Welcome from "../components/welcome/welcome";
+
+import { useState, useEffect } from "react";
+import styles from "../styles/Sugars.module.css";
+
+export default function SugarCalculator() {
+  const [ingOptions, setIngOptions] = useState([]);
+  //const [ingInst, setIngInst] = useState();
+  const [toIng, setToIng] = useState(null);
+  const [amount, setAmount] = useState<number>(1);
+  const [total, setTotal] = useState(amount);
+  const [resultMsg, setResultMsg] = useState("Solution will apper here");
+
+  useEffect(() => {
+    const getIngredients = async () => {
+      const response = await fetch(`/api/ingredients/`);
+      const ingOptions = await response.json();
+      const basicIng = ingOptions[1];
+      //const ingRate = toIng.inst;
```

Comment:
do you need this anymore?

Reply:
This is still work in progress, it will be a part of a question I would like to post. I have sorted and burnt all the rest of the comment, that in fact were not needed anymore 😊